packages/reports/src/Components/filter-component.tsx

```diff
@@ -5,7 +5,7 @@ import { connect } from 'Stores/connect';
 import CompositeCalendar from './Form/CompositeCalendar';
 import { TRootStore } from 'Stores/index';
 
-type TFilterComponent = {
+export type TFilterComponent = {
```

Comment:
Don't export the type if you only use it once. Try to use React.ComponentProps<typeof {ComponentName}>[{'value'}].